ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts

```diff
@@ -45,6 +45,9 @@ export interface Data extends RaidbossData {
 const triggerSet: TriggerSet<Data> = {
   id: 'FuturesRewrittenUltimate',
   zoneId: ZoneId.FuturesRewrittenUltimate,
+  comments: {
+    en: 'Triggers: P1-2 / Timeline: P1-5',
+  },
```

Comment:
May want to exclude this to avoid potential merge conflicts, and let @wexxlee add comments as part of #538?

Reply:
Yeah that makes sense.